common/v2/features/SendAssets/components/TransactionReceipt.tsx

```diff
@@ -37,15 +38,19 @@ export default function TransactionReceipt({ txReceipt, txConfig }: IStepCompone
     );
     if (blockNumber === 0 && txReceipt.hash) {
       const blockNumInterval = setInterval(() => {
-        getTransactionReceiptFromHash(txReceipt.hash, provider).then(transactionReceipt => {
-          if (transactionReceipt) {
-            const transactionStatus = transactionReceipt.status === 1 ? true : false;
+        getTransactionReceiptFromHash(txReceipt.hash, provider).then(transactionOutcome => {
+          if (transactionOutcome) {
+            const transactionStatus = transactionOutcome.status === 1 ? true : false;
```

Comment:
I don't understand a `status` variable that is `true` or `false`.
Maybe we can rename `status` (especially in the state) or set its value to something more explicit such as fetching, requested, pending, success, etc.

Reply:
done.